src/DependencyInjection/Configuration.php

```diff
@@ -98,7 +98,21 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->children()
                                     ->booleanNode('enabled')->defaultTrue()->end()
                                     ->arrayNode('services')
-                                        ->scalarPrototype()->end()
+                                        ->arrayPrototype()
+                                            ->children()
+                                                ->scalarNode('service')->isRequired()->end()
+                                                ->scalarNode('name')->end()
+                                                ->scalarNode('description')->end()
+                                                ->scalarNode('method')->end()
+                                                ->booleanNode('is_chain')->defaultFalse()->end()
```

Comment:
```suggestion
                                                ->booleanNode('use_chain')->defaultFalse()->end()
```
? or `can_use_chain`? `wrap_in_chain`?

Reply:
we indicate with that flag, that the configured service is a chain